pinot-core/src/test/java/com/linkedin/pinot/core/predicate/NoDictionaryInPredicateEvaluatorTest.java

```diff
@@ -225,7 +225,7 @@ public void testStringPredicateEvaluators() {
     Set<String> valueSet = new HashSet<>();
 
     for (int i = 0; i < 100; i++) {
-      stringValues[i] = RandomStringUtils.random(MAX_STRING_LENGTH);
+      stringValues[i] = RandomStringUtils.random(MAX_STRING_LENGTH).replace("\t", "");
```

Comment:
Recommend using char ('\t') instead of string

Reply:
Discussed offline